server/datastore/mysql/testing_utils.go

```diff
@@ -419,3 +419,16 @@ func DumpTable(t *testing.T, q sqlx.QueryerContext, tableName string) { //nolint
 	require.NoError(t, rows.Err())
 	t.Logf("<< dumping table %s completed", tableName)
 }
+
+// FIXME: once we introduce the ability to backfill host_uuids in the
+// mdm_windows_enrollments
```

Comment:
Can we replace this with `ds.UpdateMDMWindowsEnrollmentsHostUUID` [now](https://github.com/fleetdm/fleet/pull/14837/files#diff-aa273fb8907e3d285af4e116d2f65858f878b93a5772685c3da3601ae04b8f85R380)?